### DISCUSSION TITLE
read_json - add option to stop processing file without giving an error when parser found invalid json

### DISCUSSION THREAD

**Discussion by nuwansa** [duckdb/duckdb#8962] [Ideas]
Hi All, 
first, thank you very much for the all hard effort you guys putting to this great project. this is a feature request came to my mind when i'm working on one writer , multiple reader use case with json file.
let say we have  an append only json file with records are separated with new line.
{"name":"alex" ,"age" : 30}
{"name":"john" ,"age" : 23}
{"name":"bob" ,"age" :

if  read_json can stop processing the file when it's found the invalid json without giving an error, then we can write good one writer , multiple reader program , because readers can continue the queries without returning errors when it's found half written json record from the writer.in case of writer crash with half written json,  writer can easily recover and truncate the half written json and add new record again without disrupting the readers.

suggested option
read_json('test.json',stopAtJsonError = true) .

i think it's need better naming for the option. :)

**Comment by liquidcarbon** [comment]
What would the flag `ignore_errors` do in this case? http://duckdb.org/docs/data/json/overview

**Reply by nuwansa** [reply]
ignore_errors is doing the job. thank you

**Comment by nuwansa** [comment]
closing this since we already have the capability.
